packages/property-view/src/browser/style/property-view.css

```diff
@@ -0,0 +1,42 @@
+/********************************************************************************
+ * Copyright (C) 2020 EclipseSource and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+
+.property-view-widget {
+    padding: 5px;
+}
+
+#empty-property-view .theia-widget-noInfo {
+    padding: 8px;
+}
+
+.property-view-widget .treeContainer {
+    height: 100%;
+}
+
+.resource-tree-node-icon {
+    margin: 0 3px;
+    flex-basis: 1.5%;
+    align-self: center;
+    text-align: center;
+}
+
+.resource-tree-node-name {
+    flex-basis: 30%;
+}
+
+.resource-tree-node-property {
+    flex-basis: 70%;
+}
```

Comment:
Ideally we would have variables for these properties.
In addition, we should prefix them by `theia-`: https://github.com/eclipse-theia/theia/wiki/Coding-Guidelines#css-prefix-global-classes